wkcuber/wkcuber/download.py

```diff
@@ -0,0 +1,130 @@
+"""This module takes care of downsampling WEBKNOSSOS datasets."""
+
+from pathlib import Path
+from typing import Annotated, List, Optional
+
+import typer
+from rich import print as rprint
+
+from webknossos import BoundingBox, Dataset, Mag, webknossos_context
+from webknossos.client._defaults import DEFAULT_WEBKNOSSOS_URL
+from wkcuber.utils import parse_bbox, parse_mag
+
+app = typer.Typer(invoke_without_command=True)
+
+
+@app.callback()
+def main(
+    *,
+    path: Annotated[
```

Comment:
In the other commands you called this argument `target` or `output` which I find more descriptive instead of a generic `path` which could be confused with some input. Please make it consistent.